singer_sdk/streams/rest.py

```diff
@@ -338,7 +338,6 @@ def _request(
             else None,
         )
         self.validate_response(response)
-        logging.debug("Response received successfully.")
```

Comment:
This was an [Everything's Okay Alarm](https://simpsons.fandom.com/wiki/Everything%27s_Okay_Alarm).